Dockerfile

```diff
@@ -1,29 +1,45 @@
-from debian:buster
+FROM golang:1.16-buster
 LABEL org.opencontainers.image.source https://github.com/ovrclk/cosmos-omnibus
 
 RUN apt-get update && \
-  apt-get install --no-install-recommends --assume-yes ca-certificates curl && \
+  apt-get install --no-install-recommends --assume-yes ca-certificates curl file unzip jq && \
   apt-get clean
 
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN unzip awscliv2.zip
+RUN ./aws/install --bin-dir /usr/bin
+
+ARG PROJECT=akash
+ARG PROJECT_BIN=$PROJECT
+ARG PROJECT_DIR=.$PROJECT_BIN
+ARG VERSION=v0.12.1
+ARG REPOSITORY=https://github.com/ovrclk/akash.git
+ARG NAMESPACE
+
+ENV PROJECT=$PROJECT
+ENV PROJECT_BIN=$PROJECT_BIN
+ENV PROJECT_DIR=$PROJECT_DIR
+ENV VERSION=$VERSION
+ENV REPOSITORY=$REPOSITORY
+ENV NAMESPACE=$NAMESPACE
+
+ENV MONIKER=my-omnibus-node
+
 EXPOSE 26656 \
        26657 \
        1317  \
        9090  \
        8080
 
-ARG  PROJECT
-ARG  PROJECT_BIN
-ARG  PROJECT_VERSION
-
-ENV COSMOS_OMNIBUS_PROJECT=$PROJECT
-ENV COSMOS_OMNIBUS_PROJECT_VERSION=$PROJECT_VERSION
-
-ENV NODE_HOME=/data
-
-COPY dist/$PROJECT_BIN /bin/node
+RUN git clone $REPOSITORY /data
```

Comment:
non-blocker: Can we do a multi-stage build that produces an image that only has `/bin/$PROJECT_BIN` and `/run.sh` added to it?

Reply:
Completely agree, and I think worth including this before we merge. If only to make the image size smaller. I'll look at this later today

Reply:
That worked well - Akash image down from 2.26GB to 519MB, and it runs parts of the build in parallel too. `run.sh` does need the aws binary and a few apt packages, but this is as small as I could get it.